File: Test_for_9.py
# Python Program to Check Number is Divisible by 9

number = int(input(" Please Enter any Positive Integer : "))

if((number % 9 == 0) ):
    print("Given Number {0} is Divisible by 9".format(number))
else:
    print("Given Number {0} is Not Divisible by 9".format(number))

File: C++|Testfor9.cpp/Testfor9.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){ 		//Program to check if is divisible for 9 in C++
				
	int number,sum = 0;
	cout << "Type a Number" << "\n";
	cin >> number;
	
	do{
		sum += number % 10;
		number = number/10;

	}while(number);

	if(sum % 9)
		cout << "Not Divisible for 9\n";
	else cout << "Divisible for 9\n";

	return 0;
}

